address;source_location;insn;indent
0x18b4f3;/usr/local/src/conda/python-3.9.19/Include/object.h:628;MOV	0x8(%R15),%RAX;
0x18b4f7;/usr/local/src/conda/python-3.9.19/Python/errors.c:186;TESTB	$0x40,0xab(%RAX);
0x18b4fe;:0;JNE	18b5ab <PyErr_SetString+0x17b>;
0x18b504;/usr/local/src/conda/python-3.9.19/Include/object.h:487;TEST	%RBX,%RBX;
0x18b507;:0;JE	890e5 <PyErr_SetString.cold>;
0x18b50d;/usr/local/src/conda/python-3.9.19/Include/object.h:408;INCQ	(%RBX);
0x18b510;/usr/local/src/conda/python-3.9.19/Python/errors.c:41;XOR	%EAX,%EAX;
0x18b512;:0;JMP	18b522 <PyErr_SetString+0xf2>;
0x18b522;/usr/local/src/conda/python-3.9.19/Python/errors.c:55;MOV	0x58(%R12),%RDI;
0x18b527;/usr/local/src/conda/python-3.9.19/Python/errors.c:59;MOVQ	%RBX,%XMM0;
0x18b52c;:0;MOVQ	%R15,%XMM1;
0x18b531;/usr/local/src/conda/python-3.9.19/Python/errors.c:56;MOV	0x60(%R12),%R13;
0x18b536;/usr/local/src/conda/python-3.9.19/Python/errors.c:59;PUNPCKLQDQ	%XMM1,%XMM0;
0x18b53a;/usr/local/src/conda/python-3.9.19/Python/errors.c:57;MOV	0x68(%R12),%R14;
0x18b53f;/usr/local/src/conda/python-3.9.19/Python/errors.c:61;MOV	%RAX,0x68(%R12);
0x18b544;/usr/local/src/conda/python-3.9.19/Python/errors.c:59;MOVUPS	%XMM0,0x58(%R12);
0x18b54a;/usr/local/src/conda/python-3.9.19/Include/object.h:496;TEST	%RDI,%RDI;
0x18b54d;:0;JNE	18b58f <PyErr_SetString+0x15f>;
0x18b54f;/usr/local/src/conda/python-3.9.19/Include/object.h:496;TEST	%R13,%R13;
0x18b552;:0;JNE	18b599 <PyErr_SetString+0x169>;
0x18b554;/usr/local/src/conda/python-3.9.19/Include/object.h:496;TEST	%R14,%R14;
0x18b557;:0;JNE	18b6b3 <PyErr_SetString+0x283>;
0x18b55d;/usr/local/src/conda/python-3.9.19/Include/object.h:496;TEST	%RBP,%RBP;
0x18b560;:0;JE	18b56c <PyErr_SetString+0x13c>;
0x18b562;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%RBP);
0x18b566;:0;JE	89145 <PyErr_SetString.cold+0x60>;
0x18b56c;/usr/local/src/conda/python-3.9.19/Python/errors.c:244;MOV	0x28(%RSP),%RAX;
0x18b571;:0;SUB	%FS:0x28,%RAX;
0x18b57a;:0;JNE	18b712 <PyErr_SetString+0x2e2>;
0x18b58f;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%RDI);
0x18b592;:0;JNE	18b54f <PyErr_SetString+0x11f>;
0x18b599;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%R13);
0x18b59d;:0;JNE	18b554 <PyErr_SetString+0x124>;
0x18b59f;/usr/local/src/conda/python-3.9.19/Objects/object.c:2205;MOV	0x8(%R13),%RAX;
0x18b5a3;/usr/local/src/conda/python-3.9.19/Objects/object.c:2209;MOV	%R13,%RDI;
0x18b5a6;:0;CALLQ	0x30(%RAX);
0x18b5a9;/usr/local/src/conda/python-3.9.19/Objects/object.c:2209;JMP	18b554 <PyErr_SetString+0x124>;
0x18b5ab;/usr/local/src/conda/python-3.9.19/Python/errors.c:187;MOV	%R15,%RDI;
0x18b5ae;:0;CALL	16ac10 <PyException_GetTraceback>;
0x18b5b3;/usr/local/src/conda/python-3.9.19/Include/object.h:487;TEST	%RBX,%RBX;
0x18b5b6;:0;JE	18b5bb <PyErr_SetString+0x18b>;
0x18b5b8;/usr/local/src/conda/python-3.9.19/Include/object.h:408;INCQ	(%RBX);
0x18b5bb;/usr/local/src/conda/python-3.9.19/Python/errors.c:46;TEST	%RAX,%RAX;
0x18b5be;:0;JE	18b510 <PyErr_SetString+0xe0>;
0x18b5c4;/usr/local/src/conda/python-3.9.19/Python/errors.c:46;LEA	0x214cb5(%RIP),%RCX;
0x18b5cb;:0;CMP	%RCX,0x8(%RAX);
0x18b5cf;:0;JE	18b522 <PyErr_SetString+0xf2>;
0x18b5d5;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%RAX);
0x18b5d8;:0;JE	891ba <PyErr_SetString.cold+0xd5>;
0x18b5de;:0;XOR	%EAX,%EAX;
0x18b5e0;/usr/local/src/conda/python-3.9.19/Include/object.h:422;JMP	18b522 <PyErr_SetString+0xf2>;
0x18b6b3;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%R14);
0x18b6b6;:0;JNE	18b55d <PyErr_SetString+0x12d>;
0x18b6bc;/usr/local/src/conda/python-3.9.19/Include/object.h:430;MOV	%R14,%RDI;
0x18b6bf;:0;CALL	1296a0 <_Py_Dealloc>;
0x18b6c4;:0;JMP	18b55d <PyErr_SetString+0x12d>;
0x18b712;/usr/local/src/conda/python-3.9.19/Python/errors.c:244;CALLQ	0x1f3050(%RIP);
0x18b718;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%R15);
0x18b71b;:0;JNE	18b4f3 <PyErr_SetString+0xc3>;
